server/datastore/mysql/migrations/tables/20240813103800_SupportMultipleABMTokens.go

```diff
@@ -154,9 +152,9 @@ INSERT INTO abm_tokens
 		ipados_default_team_id
 	)
 VALUES
-	('', '', ?, DATE('2000-01-01'), ?, ?, NULL, NULL)
+	('', '', ?, DATE('2000-01-01'), ?, ?, ?, ?)
 `
-	res, err := tx.Exec(insABM, abmTermsExpired, token, defaultTeamID)
+	res, err := tx.Exec(insABM, abmTermsExpired, token, defaultTeamID, defaultTeamID, defaultTeamID)
```

Comment:
Following a convo with Noah, we do want that AppleBMDefaultTeam to apply to all platforms.